client/src/game/shapes/text.ts

```diff
@@ -3,22 +3,26 @@ import { BoundingRect } from "@/game/shapes/boundingrect";
 import { Shape } from "@/game/shapes/shape";
 import { ServerText } from "../comm/types/shapes";
 import { rotateAroundPoint } from "../utils";
+import invert from "invert-color";
 
 export class Text extends Shape {
     type = "text";
     text: string;
     font: string;
+    readable = false;
```

Comment:
I'm a bit conflicted on this.  The first issue is that this is not synced to the server, which is fine for a ruler on your own client, but will show up differently on another client for example.

Additionally this conflicts with text objects that actually do set a strokeColour.
What's wrong with setting the stroke colour on the text object for the ruler ? If you really want the shadow effect you can set it to the same colour as the stroke colour if it is set.

What do you think ?

Reply:
Yep, text with strokeColour will conflict with this, I didn't realize there is before.